### PR TITLE
Contains the content updates to introduce the debugging charm-build command content

### PR DESCRIPTION
**Pull request by chuckbutler**

### CODE REVIEW

File: src/en/developer-debug-layers.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- config.yaml
+- metadata.yaml
+
+These two files are special cases and are instead merged into a single file in
+the top most charm layer. Allowing the developer to define only fragments
+of information in their respective files.
+
+Given this behavior, it can be extremely difficult to assemble from layers,
+charms written in other frameworks/technologies other than Reactive. As Hooks
```
Comment (mbruzek) [inline src/en/developer-debug-layers.md]: "Hooks" and "hooks" used in the same sentence.  Why.

Comment (mbruzek) [inline src/en/developer-debug-layers.md]: :cop: Grammar police

**Comment by mbruzek** [comment]
+1 LGTM

File: src/en/developer-debug-layers.md (hunk 1 above)
Comment (chuckbutler) [inline src/en/developer-debug-layers.md]: :no_bicycles:

**Pull request merged**